files/build_templates/mgmt-framework.timer

```diff
@@ -8,4 +8,4 @@ OnBootSec=3min 30 sec
 Unit=mgmt-framework.service
 
 [Install]
-WantedBy=timers.target sonic.target
+WantedBy=timers.target sonic.target delay.target
```

Comment:
Maybe the target should have a more descriptive name? Maybe `sonic-delayed.target`?

Reply:
Updated.